### ISSUE TITLE
Fence 1.00->1.10 break compatibility (with an i)

### ISSUE THREAD

**Issue by Petervos2018** [Stubs42/OrangeLine#5]
I know it's a beta but shouldn't it at least try to recall the previous settings ?
It goes from : A#1-A2 to C#4-C5 after recall.
![Fence100](https://user-images.githubusercontent.com/46076474/68536523-9fb81900-0354-11ea-8c48-d8dca32af030.png)

to

![Fence110_cr](https://user-images.githubusercontent.com/46076474/68536524-a777bd80-0354-11ea-9ba0-b8d92be05ec2.png)

Greetings Peter.

**Comment by Stubs42** [comment]
Hi Peter,
As you might know, I'm just started with module development and have to learn a lot. 1.0 Fence used hidden parameters for storage. This was changed to json storage. I know that compability is an important topic. As soon I remove the Beta mark, compatibility will be targeted with high priority. With switching to json storage, this will be much easier in the future. I also listen to any constructive feedback of eary adopters to make the best out of a module. This might also cause changes in functionality and behavior of my modules as long they are marked beta. Nevertheless I will also put more effort in compatibility of beta modules in the future. 
Sry for your inconvenience.

**Comment by Petervos2018** [comment]
Hi Dieter,
No problem, I just noticed that my track was sounding weird and all I did was update your module.
I went to github to see what changed. And there was no mention that the old patches are not compatible with the new version, only that "Fence was going through massive refactoring and got some nice new features" and I didn't really know what that meant.
I'm on a PC so I'm used to compatibility, unlike the Apple people ;)

Is there any logic to the update? 
I mean add 9 semitones and subtract 3 octaves and you have the old settings or is it random?
Love this plug in, I used to use NYSTHIs octave fold but yours ~~is~~ was more accurate.
Thanks for answering,
Greetings Peter.
